apis/duck/v1alpha1/channelable_types_test.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/knative/pkg/apis/duck"
+)
+
+func TestChannelImplementsChannelable(t *testing.T) {
```

Comment:
Any reason not to move all of these new tests into a single file within this package, that just checks that all the given types satisfy the expected duck types? It could be a pretty simple table-driven test.

Reply:
@ImJasonH didn't thought of that initially, work for me :wink: